user_guide_src/source/cli/cli_commands/006.php

```diff
@@ -2,6 +2,6 @@
 
 try {
     // ...
-} catch (\Exception $e) {
+} catch (Exception $e) {
```

Comment:
Is it better to set `leading_backslash_in_global_namespace` to true for User Guide?
See https://cs.symfony.com/doc/rules/import/fully_qualified_strict_types.html

If we do so, the leading `\` will not be removed, and if it is not in the current sample code it will be added.

Reply:
leading backlash has no effect when in global namespace, right? so adding it is superfluous.

Reply:
It has no effect in PHP syntax, but actually most code will be written in some namespace like `App\Controllers`,
So when there is no leading backslash,  if a user copies and pastes the sample code, the code will not work unless they import the class.

Is it better that sample code should work as it is as possible when a dev copies and pastes?

Reply:
In such cases, it may be better to add namespaces for those files.

Reply:
IMO the code copied from the user guide should work out of the box. Less experienced users may not know they have to import the class. If there is a way to automatically add an import to our code examples then that would be fine. Otherwise, I would leave it as it is.

Reply:
> the code copied from the user guide should work out of the box.

I agree with this.

`import_symbols` option imports symbols when the classname in the code is namespaced (containing `\`).
But `Exception` in the above sample is not imported.